zql/zql.peg

```diff
@@ -497,36 +497,23 @@ NotExpression
   / CallExpression
 
 
-#ifdef notyet
-
 CallExpression
-  = fn:FunctionName __ "(" __ args:ArgumentList __ ")" {
-        RETURN(XXX)
+  = fn:FunctionName __ "(" args:ArgumentList ")" {
+        RETURN(makeFunctionCall(fn, args))
     }
   / DereferenceExpression
 
 FunctionName
-  = first:FunctionNameComponent
-    rest:("." n:FunctionNameComponent { RETURN(n) })* {
-        RETURN(XXX)
-    }
-
-FunctionNameComponent
   = FunctionNameStart FunctionNameRest* { RETURN(TEXT) }
 
 FunctionNameStart = [A-Za-z]
-FunctionNameRest = FunctionNameStart / [0-9]
+FunctionNameRest = FunctionNameStart / [.0-9]
```

Comment:
I forgot to add this comment yesterday, and this can be sorted out later, but do we want function calls like `Math..sqrt(x)` to be syntactically valid, correct? (I'd think not).

Reply:
Good question, I had started to write more complicated grammar rules earlier that would have prevented this earlier (https://github.com/brimsec/zq/blob/2d8a04c3d6aae61eb1d53c29fc093d64edd15c91/zql/zql.peg#L508-L515) but I wanted to keep the grammar simple.  If we never create functions with such names, then this would be a compilation error (as opposed to a parse error).  Do you think that's adequate?

Reply:
Yep, I think this is totally adequate for now. Given that our syntax errors aren't currently very helpful, it's actually more useful to let `Math..sqrt()` all the way through to the function lookup check. Maybe we can revisite this when we get around to improving syntax errors with locations etc...